File: Practice/11/C++/stepen.cpp
#include <math.h>;
#include <stdio.h>;
using namespace std;
int number, i, mnojitel;
double stepen;
int main() {
	setlocale(LC_CTYPE, "rus");
	cout << "Введите число ";
	cin >> number;
	cout << "Введите степень ";
	cin >> stepen;
	if (stepen == 0) {
		cout << 1;
	}
	else if (stepen == 0.5) { number = sqrt(number); }
	else {
		mnojitel = number;
		for (i = 2; i <= stepen; i++) {
			number = number * mnojitel;
		}
		cout << number;
	}
}

File: Practice/11/Python/Stepen.py
print("Введите число и степень")
number=float(input())
stepen=float(input())
mnojitel=float(number)
Bmnojitel = 1
if stepen == 1:
    print(c)
elif stepen == 0:
    print(1)
elif stepen>0:
    while Bmnojitel < stepen:
        Bmnojitel = Bmnojitel + 1
        number = number * Bmnojitel
    print(number)
else:
    d = float(1/number)
    while Bmnojitel > stepen:
        Bmnojitel = Bmnojitel - 1
        number = number * d
    print(number)